apps/comments-ui/src/utils/hooks.ts

```diff
@@ -57,3 +57,18 @@ export function useRelativeTime(dateString: string) {
         return formatRelativeTime(dateString, t);
     }, [dateString]);
 }
+
+export const useHighlightComment = () => {
+    const {dispatchAction} = useAppContext();
+
+    const highlightComment = (commentId: string) => {
+        dispatchAction('highlightComment', {commentId});
+
+        // clears the highlight after 2.5 seconds
+        setTimeout(() => {
+            dispatchAction('highlightComment', {commentId: null});
+        }, 2500);
+    };
+
+    return highlightComment;
+};
```

Comment:
Can the `setTimeout` live directly inside the `highlightComment` action to avoid this extra hook? That wouldn't be too much different to how we're planning to make actions update state immediately then re-update once the API request finishes